### PR TITLE
Add Atom to text Editors

### PR DESCRIPTION
**Pull request by yparks**
- **Section modified:**  Text editors resource to the readme file
- **Graduating class:** 2019

Why:
Atom makes collaborating on code just as easy as it is to code alone.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Are you a Hackbright alumna or student?
   - [Babel](https://packagecontrol.io/packages/Babel) &mdash; Syntax definitions for JSX and modern JS
 - [Brackets](http://brackets.io/) &mdash; Text editor aimed toward web design
 - [Neovim](https://neovim.io/) &mdash; Vim, but it uses Python as its scripting language
+-[Atom](https://atom.io/) &mdash; A free and open-source text and source code editor
```
Comment (mearajennifer) [inline readme.md]: Add a space after the dash and before Atom (see entries above).

**Review by mearajennifer — CHANGES_REQUESTED** [review]
Thank you! 🐰

File: readme.md (hunk 1 above)
Comment (yparks) [reply readme.md]: Just changed it.

**Review by mearajennifer — APPROVED** [review]
Approved! 🦄

**Pull request merged**